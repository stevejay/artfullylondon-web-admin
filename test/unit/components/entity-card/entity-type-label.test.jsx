import React from 'react'
import { shallow } from 'enzyme'

import EntityCardEntityTypeLabel
  from '_src/components/entity-card/entity-type-label'

it('should render correctly', () => {
  const wrapper = shallow(
    <EntityCardEntityTypeLabel
      entity={{
        entityType: 'venue',
        entityTypeLabel: 'The Label'
      }}
    />
  )

  expect(wrapper).toMatchSnapshot()
})
