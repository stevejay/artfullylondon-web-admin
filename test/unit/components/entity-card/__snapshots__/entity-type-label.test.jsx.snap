// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="src-components-entity-card-entity-type-label__venue src-components-entity-card-entity-type-label__label"
>
  The Label
</div>
`;
